### PR TITLE
menus/index change suggestions

### PR DESCRIPTION
**Pull request by yatil**
Proposal of some changes for PR #427.

- Add an “are” in the first sentence to make it easier to read.
- Use label instead of identify as I think identify is pretty abstract.
- Add “special” to the Application Menus sentence to give emphasis to the fact that this is additional work.
- Under “Why is this important?” some changes:
  - Remove “, individual web pages, and multi-step processes, such as a check-out form” as we don’t have such examples in the tutorial anymore (sorry for the confusion)
  - Reworded “Application menus provide key functionality of the application.” to “Application menus provide access to key functionality of an application.” – The previously used sentence sounded too much like “Application menus are menus for applications.” ;-)
  - Added applications to “Thus menus are critical parts of web pages and applications, and require particular attention during design and development.”

### CODE REVIEW

File: source/menus/index.html.erb.md [hunk 1]
```diff
@@ -26,19 +26,19 @@ wcag_techniques:
...
 
-* **[Structure](structure.html):** Mark up menus in a way that reflects their structure and that identifies them as such.
+* **[Structure](structure.html):** Mark up menus in a way that reflects their structure and labels them.
 
 * **[Styling](styling.html):** Use commonly recognized design patterns to distinguish menus and the state of menu items.
 
 * **[Fly-out Menus](flyout.html):** Ensure fly-out (drop-down) sub-menus can be used appropriately by mouse and keyboard.
 
-* **[Application Menus](application-menus.html):** Add markup and keyboard behavior to resemble desktop application menus.
+* **[Application Menus](application-menus.html):** Add special markup and keyboard behavior to resemble desktop application menus.
```
Comment (nitedog) [inline source/menus/index.html.erb.md]: Not fond of "special" but don't feel strongly.

Comment (nitedog) [inline source/menus/index.html.erb.md]: **IMPORTANT** in some tutorials we use "mark-up" (with hyphen) and in others "markup". Please check for consistency across all tutorials.

**Comment by nitedog** [comment]
Looks all good to me!

File: source/menus/index.html.erb.md (hunk 1 above)
Comment (yatil) [inline source/menus/index.html.erb.md]: Would additional be better? Or what about the following [synonyms](http://www.thesaurus.com/browse/special?s=ts):

* certain
* different
* particular (Eric likes that one)
* specific

Comment (yatil) [inline source/menus/index.html.erb.md]: I’ve opened #465.

Comment (nitedog) [inline source/menus/index.html.erb.md]: Happy to go with particular. Specific also works. Add additional sounds odd :-) Can also live without anything - just "add markup ... to resemble desktop" works fine for me.

Comment (yatil) [inline source/menus/index.html.erb.md]: I want to make sure that it’s clear that there are two different markup approaches when using application menus. I’ll use particular. :-)

**Pull request merged**